src/Core/src/Platform/iOS/MauiSwipeView.cs

```diff
@@ -807,20 +807,19 @@ void SwipeToThreshold(bool animated = true)
 				Animate(completeAnimationDuration, 0.0, UIViewAnimationOptions.CurveEaseIn,
 					() =>
 					{
-						_swipeOffset = GetSwipeThreshold();
-						double swipeThreshold = _swipeOffset;
+						_swipeOffset = Math.Abs(GetSwipeThreshold());
+						if(_swipeDirection == SwipeDirection.Left || _swipeDirection == SwipeDirection.Up)
+							_swipeOffset = -_swipeOffset;
```

Comment:
Could you include a comment here? Something like "If the user swiped left or up, we need a negative offset to move content in the correct direction on the screen."